vignettes/VectorControl_Bednets.Rmd

```diff
@@ -0,0 +1,129 @@
+
+The default setting is to model all mosquito species together. 
+```{r}
+simparams$species
+simparams$species_proportions
+```
+  
+## Simulation
+
+Having established a base set of parameters, we can now create a copy of this parameter list and update it to specify a net distribution campaign. In the example below, we distribute bed nets to 50% of the population every three years. It is possible to change the characteristics of the bed nets for each distribution timestep if different types of bed nets are distributed that have different insecticides, different levels of insecticide resistance, etc. This can be done by modifying the matrices for `dn0`, `rn`, `rnm`, and `gamman`. Because we are using the default proportions of mosquito species and there are two timesteps when nets are distributed, the matrices for `dn0`, `rn`, and `rnm` have 1 column and 2 rows.
+
+The parameter values for pyrethroid-only and pyrethroid-PBO nets at various resistance levels can be found in Table S1.3 in the Supplementary Appendix 2 of [Sherrard-Smith, et al., 2022](https://doi.org/10.1016/S2542-5196(21)00296-5).
+
+  
+```{r, fig.align="center", fig.height=4, fig.width=6}
+bednetstimesteps <- c(1, 4) * year # The bed nets will be distributed at the end of the first and the 4th year. 
+
+bednetparams <- set_bednets(
+  simparams,
+  timesteps = bednetstimesteps,
+  coverages = c(.5, .5),  # Each round is distributed to 50% of the population.
+  retention = 5 * year, # Nets are kept on average 5 years
+  dn0 = matrix(c(.533, .533), nrow = 2, ncol = 1), # Matrix of death probabilities for each mosquito species over time
+  rn = matrix(c(.56, .56), nrow = 2, ncol = 1), # Matrix of repelling probabilities for each mosquito species over time
+  rnm = matrix(c(.24, .24), nrow = 2, ncol = 1), # Matrix of minimum repelling probabilities for each mosquito species over time
+  gamman = rep(2.64 * 365, 2) # Vector of bed net half-lives for each distribution timestep
+)
+
+output <- run_simulation(timesteps = sim_length, parameters = bednetparams)
+```
+
+### Plot prevalence
+```{r, fig.align="center", fig.height=5, fig.width=7}
+plot_prev()
+```
+  
+## Comparing input coverage and population bed net usage 
+
+It is important to understand the difference between the input `coverages` argument of `set_bednets()` and the resulting population bed net usage over time in the model. When we set coverages in the above example to 0.5, we are telling the model to distribute bed nets to 50% of the population at year one and to 50% of the population at year 4. However, the level of average bed net usage is not necessarily equal to 50%. Between these time points, bed net use will slowly decline over time (in reality this decline stems from things like disuse, holes in nets, lost nets, etc.).
```

Comment:
Indicate that we are telling the model to distribute bed nets to a _random_ 50% of the population at years 1 and 4. Then will hopefully make more sense why the average usage is above 50% in the plot below.